src/frontend/components/property-type/boolean/edit.tsx

```diff
@@ -29,6 +29,7 @@ const Edit: React.FC<EditPropertyProps> = (props) => {
         {...property.props}
       />
       <PropertyLabel property={property} props={{ inline: true }} />
+      {property.description && <FormMessage>{property.description}</FormMessage>}
```

Comment:
I think this should go through i18n to support multi language, might also be a good idea to create a separate component for description.

Another suggestion is to use design system's tooltip component for that: https://storybook.adminjs.co/?path=/story/designsystem-atoms-tooltip--default

and use Carbon Design's `Help` icon as hoverable element: https://www.carbondesignsystem.com/guidelines/icons/library/